extension/src/time_vector/pipeline.rs

```diff
@@ -477,7 +455,7 @@ mod tests {
                 (ts:\"2020-04-10 00:00:00+00\",val:5.8046),\
                 (ts:\"2020-04-14 00:00:00+00\",val:7.195),\
                 (ts:\"2020-04-20 00:00:00+00\",val:10.0221)\
-            ]");
+            ],is_sorted:true)");
```

Comment:
This is interesting: I was deliberate when implementing the timevector text format so that it would _only_ contain the data, and not the metadata such as the `num_points` or `is_sorted`. I did this because this seemed like one of the few types where working with the text form of the type could be useful, so keeping only the data minimized the possibility of data validation errors. That said,I guess none of this matters yet, so we can punt on this until later a long a we record the decision.

Reply:
I still think it makes sense to call `unnest` to get a readable form.  Even without the metadata the points aren't that clean.